tests/artist-read.test.js

```diff
@@ -0,0 +1,63 @@
+const {expect} = require('chai')
+const request = require('supertest')
+const db = require('../src/db')
+const app = require('../src/app')
+
+describe('Read Artists',()=>{
+  let artists
+  beforeEach(async ()=>{
+    const responses = await Promise.all([
+      db.query('INSERT INTO Artists (name,genre) VALUES ($1,$2) RETURNING *',[
+        'Tame Impala',
+        'rock',
+      ]),
+      db.query('INSERT INTO Artists (name,genre) VALUES ($1,$2) RETURNING *',[
+        'Queen',
+        'rock',
+      ]),
+      db.query('INSERT INTO Artists (name,genre) VALUES ($1,$2) RETURNING *',[
+        'Stevie Wonder',
+        'soul',
+      ]),
+    ])
+
+    artists=responses.map(({rows})=>rows[0])
+  })
+
+  describe('GET /artists',()=>{
+    it('returns all artist records in the database',async()=>{
+      const{status,body}=await request(app).get('/artists').send()
+
+      expect(status).to.equal(200)
+      expect(body.length).to.equal(3)
+
+      body.forEach((artistRecord)=>{
+        const expected = artists.find((a)=>a.id === artistRecord.id)
+
+        expect(artistRecord).to.deep.equal(expected)
+      })
+    })
+  })
+
+
+
+describe('GET /artists/{id}',()=>{
+  it('returns the artist with the correct id',async()=>{
+    const {status,body}=await request(app).get(`/artists/${artists[0].id}`).send()
+
+    expect(status).to.equal(200)
+    expect(body).to.deep.equal(artists[0])
+  })
+
+it('returns a 404 if the artist does not exist',async()=>{
+  const {status,body}=await request(app).get(`/artists/999999999`).send()
+
+  expect(status).to.equal(404)
+  expect(body.message).to.equal('artist with 999999999 does not exist')
+})
+
+
+})
+})
+
```

Comment:
It would be impractical for me to correct all the formatting issues throughout the work here, when you could do this yourself very easily by using the format on save feature in vscode. **Please** look into adding this to your dev environment, it takes a couple of minutes to set up and your work will instantly be more readable with the proper indentation and formatting, and making your (and anyone elses) life easier when it comes to reading your work.